### ISSUE TITLE
No recrutable AI in Living Quater on "Conflict - Everon (RHS)"

### ISSUE THREAD

**Issue by DeadPig** [RHSMODS/statusquo#1013]
At the Living Quater, after opening the "Request Personnel" build menu, there are no selectable entries/soldiers to request/spawn.
The tiles to select them are just empty.

Used Version:
Arma Reforger Experimental: 1.2.1.127
RHS - Status Quo: 0.10.3742 (Client+Server) / Dedicated Server
(No other mods installed/active)
![20241120162547_1](https://github.com/user-attachments/assets/48ef511d-393d-4965-8f5f-40969fba267f)

**Comment by MasonicVenture1** [comment]
Any ideas of when this bug will be fixed? i was really enjoying solo vs my buddy until we found this

**Comment by reyhard** [comment]
This might need modding vanilla scripts - other options require making tons of duplicated prefabs (which we don't want and don't have time to) or making a structure, which would break vanilla - this could be fine for mods which are single scenarios only but in case of RHS a whole that would be a no go.

TLDR:
not soon

**Comment by DeadPig** [comment]
> should be fixed in dev

Hoi!
Sry for late confirmation.
Yes, seems to be working on my server too using Dev-Branch.
Thx!
